composer.json

```diff
@@ -25,16 +25,16 @@
     ],
     "require": {
         "php": ">=5.3.2",
-        "symfony/framework-bundle": "~2.2",
+        "symfony/framework-bundle": "~2.6",
         "doctrine/dbal": "~2.3",
         "jdorn/sql-formatter": "~1.1",
-        "symfony/doctrine-bridge": "~2.2",
+        "symfony/doctrine-bridge": "~2.6",
```

Comment:
you don't have anything depending on symfony/doctrine-bridge 2.6. You have things depending on symfony/dependency-injection 2.6